README.md

```diff
@@ -0,0 +1,8 @@
+# Meeting room dashboard
+
+## Docker usage example
+### Build
+`docker build . --tag meeting-room-dashboard:latest`
```

Comment:
always attach a versioned tag also like: `docker build . --tag meeting-toom-dashboard:v8 --tag meeting-toom-dashboard:latest`

you can't update a `docker service` from `latest` to `latest` with `docker service update --image=meeting-toom-dashboard:latest meeting-toom-dashboard`

use the commit hash, a build number as version and deploy a specific version.